scripts/GEAR/convert_GEAR_beam.py

```diff
@@ -19,6 +20,7 @@
         T,    
         )
 from pangeo_forge_recipes.types import Indexed
+logging.basicConfig(level=logging.INFO)
```

Comment:
Fine for now but it would be nice to include the timestamps at the start of each log message (I don't think the basic config does this by default)

Reply:
[Added](https://github.com/NERC-CEH/dri_gridded_data/pull/29/commits/cca194b16738e1407838cd3bf2a4d9fe323624b8)